packages/components/src/components/checkbox/checkbox.scss

```diff
@@ -42,7 +42,7 @@
 
     &__label {
         &--error {
-            color: var(--text-loss-danger);
+            color: var(--text-loss-danger) !important;
```

Comment:
why is this `!important` required now?
is it a good practice?

Reply:
this is a very valid point. I believe mostly we should avoid it, but for default components such as checkbox, inputs etc, the default functionality have to be consistent.
the same we have in styles for example for input, input-field components,